.env_sample

```diff
@@ -0,0 +1 @@
+GOOGLE_API_KEY="secret api key"
```

Comment:
There are more keys :) We need all of them.

Reply:
Ok, if you fix merge conflict all will be ok.

Reply:
Yep, I started this branch before you did dotenv.